src/components/FeatureViews/Search/CoursesCards.js

```diff
@@ -0,0 +1,100 @@
+import { connect } from "react-redux";
+import React, { useState } from "react";
+import BackButton from "../../BackButton";
+import Grid from "@material-ui/core/Grid";
+import { Card, Paper, Typography, Grow } from "@material-ui/core";
+
+import Chip from "@material-ui/core/Chip";
+import CardActions from "@material-ui/core/CardActions";
+import { withRouter } from "react-router-dom";
+
+
+
+function CourseCards(props) {
+    const [value, setValue] = useState(0);
+    const [userList, setUserList] = useState([])
+    const [viewToggle, setViewToggle] = useState(true);
+
+    const goToRoute = (route) => {
+        props.history.push(props.match.url + route);
+    }
```

Comment:
You can make this a hook in a separate file, and then import it in to the places where it is used.